packages/core-utils/src/time.js

```diff
@@ -88,34 +127,24 @@ export function formatSecondsAfterMidnight(seconds, timeFormat) {
  */
 export function getUserTimezone() {
```

Comment:
It'd be nice to see a unit test for this. And also to know what happens for browsers that don't support this API. Do we get a white screen of death? I'd prefer to avoid that if possible! Also the comment probably needs updating.

Reply:
Have done some testing with this on browser stack and you get a reference error (since `Intl` isn't in `window`). Have added a UTC+0 default for now.